openfl/experimental/workflow/component/aggregator/aggregator.py

```diff
@@ -198,6 +198,9 @@ async def run_flow(self) -> FLSpec:
         Returns:
             flow (FLSpec): Updated instance.
         """
+        # As an aggregator is created before a runtime, set prohibited data types for the runtime
+        # before running the flow.
+        self.flow.runtime.prohibited_data_types = FederatedRuntime.prohibited_data_types
```

Comment:
It would be helpful to follow the Metaflow pattern in the next function for `include`/`exclude`; we need to have `allowed_data_types` as well. I expect it will be far easier to specify what's allowed vs. trying to capture all the possible types that are not, i.e. `allowed_data_types=[np.ndarray]`

Reply:
AFAIS currently we pass 'str' type as the 'next' function argument in notebooks:
  self.next( self.aggregated_model_validation, foreach="collaborators")
Following your suggestion, we are also prohibiting numeric types, even though there is no reason for doing so.